zookeeper-server/src/main/java/org/apache/zookeeper/server/backup/RestoreFromBackupTool.java

```diff
@@ -546,6 +581,42 @@ private boolean validateLogRange() {
     return true;
   }
 
+  /**
+   * Check if the specified snap dir and data dir already have files inside.
+   * If so, ask user to confirm if they want to overwrite these two directories with restored files.
```

Comment:
Let's clarify the meaning of "overwrite" here in the JavaDoc - will existing files be wiped out clean and the path be populated with restored files?